tests/plugins/balloontoolbar/positioning.js

```diff
@@ -229,6 +229,181 @@
 
 				assert.isTrue( markerElement.equals( spy.args[ 0 ][ 0 ] ) );
 			} );
+		},
+
+		// #1653
```

Comment:
Please, update with `(#1653)` to our adopted code style.

Reply:
I feel like we don't have defined code style for it, as there is plenty of examples of both approaches with and without parentheses. However, I don't mind to change it.